python/services/notify_service.py

```diff
@@ -27,20 +22,31 @@ def send_first_email(self, email_address: str, login_date: str):
 
     def send_reminder_email(self, email_address: str, login_date: str):
         personalisation = {"login_date": login_date}
+
+        template_id = ""
+        if self.organisation_name == MINISTRY_OF_JUSTICE:
+            template_id = "7405b6f8-9355-4572-8b8c-c73bc8cdee3c"
+
+        if self.organisation_name == MOJ_ANALYTICAL_SERVICES:
+            template_id = "13863d96-7986-4c3b-967e-3123a6773896"
+
```

Comment:
Is it worth having the same exception here? Although I do appreciate it wasn't there before! 😁